mne/beamformer/tests/test_external.py

```diff
@@ -0,0 +1,93 @@
+# preparing everything:
+import mne
+import numpy as np
+import os.path as op
+from scipy.io import savemat
+
+from mne.datasets import testing
+from mne.beamformer import make_lcmv, apply_lcmv
+from mne.beamformer.tests.test_lcmv import _get_data
+
+
+data_path = testing.data_path(download=False)
+fname_raw = op.join(data_path, 'MEG', 'sample', 'sample_audvis_trunc_raw.fif')
+fname_cov = op.join(data_path, 'MEG', 'sample', 'sample_audvis_trunc-cov.fif')
+fname_fwd = op.join(data_path, 'MEG', 'sample',
+                    'sample_audvis_trunc-meg-eeg-oct-4-fwd.fif')
+fname_fwd_vol = op.join(data_path, 'MEG', 'sample',
+                        'sample_audvis_trunc-meg-vol-7-fwd.fif')
+fname_event = op.join(data_path, 'MEG', 'sample',
+                      'sample_audvis_trunc_raw-eve.fif')
+fname_label = op.join(data_path, 'MEG', 'sample', 'labels', 'Aud-lh.label')
+
+reject = dict(grad=4000e-13, mag=4e-12)
+
+# where FieldTrip data should be saved to / imported from
+ft_data_path = ' '
```

Comment:
@larsoner where should the data live?

Reply:
I would make it `MNE-testing-data/fieldtrip/*`